components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/RecoveryScenarios.java

```diff
@@ -28,10 +28,13 @@ public enum RecoveryScenarios {
     USERNAME_RECOVERY,
     SELF_SIGN_UP,
     ASK_PASSWORD,
```

Comment:
Let's add a comment ASK_PASSWORD is related to email link flow.

Reply:
Fixed by - https://github.com/wso2-extensions/identity-governance/pull/998/commits/0ec71b0e4235f72669815f3314c3aeedad5324ec